server/src/main/java/io/druid/server/initialization/jetty/ChatHandlerServerModule.java

```diff
@@ -50,15 +54,28 @@
   @Override
   public void configure(Binder binder)
   {
-    /** If "druid.indexer.task.chathandler.port" property is set then we assume that a
-     * separate Jetty Server with it's own {@link ServerConfig} is required for ingestion apart from the query server
-     * otherwise we bind {@link DruidNode} annotated with {@link RemoteChatHandler} to {@literal @}{@link Self} {@link DruidNode}
-     * so that same Jetty Server is used for querying as well as ingestion
+    Jerseys.addResource(binder, ChatHandlerResource.class);
+
+    if (properties.containsKey(MAX_CHAT_REQUESTS_PROPERTY)) {
+      final int maxRequests = Integer.parseInt(MAX_CHAT_REQUESTS_PROPERTY);
```

Comment:
does it need a parse catcher? I think it's ok for this to blow up and let Guice report the error if you pass something that isn't a number